README.md

```diff
@@ -105,6 +126,19 @@ ansible-playbook -vv -i "localhost," --connection=local local-test-operator.yml
     -e production_quay_namespace="certified-operators" \
     -e operator_dir="${OPERATOR_DIR}"
 ```
+##### If we want to run the full ISV operator testing with image pull secrets
```

Comment:
why is this only for ` ISV operators only`?
If you using the playbooks in standalone mode with a community operator...would image pull secrets not work?

Reply:
I would re-phrase like this:

`If we want to run full operator testing with image pull secrets and certified-operators`